app/_locales/th/messages.json

```diff
@@ -164,10 +164,29 @@
     "message": "แก้ไขผู้ติดต่อ"
   },
   "endOfFlowMessage1": {
-    "message": "คุณผ่านการทดสอบแล้ว กรุณารับผิดชอบในการรักษา Seed Phrase ให้ปลอดภัย"
+    "message": "คุณผ่านการทดสอบแล้ว โปรดรักษา seedphrase ของคุณให้ปลอดภัย ถือเป็นความรับผิดชอบของคุณ!"
```

Comment:
Could you update this to say secret recovery phrase instead of seed phrase?

Reply:
Of course, I replace 'seed phrase' with 'secret recovery phrase' and the translation quality has been improved in fce66895e8393a88364f4b070cfebbd506c03d1e.

Thank you.